client/payment-details/payment-method/affirm/index.tsx

```diff
@@ -32,20 +34,36 @@ const formatPaymentMethodDetails = ( charge ) => {
 /**
  * Placeholders to display while loading.
  */
-const paymentMethodPlaceholders = {
+const paymentMethodPlaceholders: PaymentMethodDetails = {
 	id: 'id placeholder',
 	name: 'name placeholder',
 	email: 'email placeholder',
 	formattedAddress: 'address placeholder',
 };
 
-const CardDetails = ( { charge = {}, isLoading } ) => {
-	const details =
+/**
+ * Props interface for AffirmDetails component
+ */
+interface AffirmDetailsProps {
+	charge?: Charge;
+	isLoading: boolean;
+}
+
+const AffirmDetails: React.FC< AffirmDetailsProps > = ( {
+	charge,
+	isLoading,
+} ) => {
+	const details: PaymentMethodDetails | typeof paymentMethodPlaceholders =
```

Comment:
```suggestion
	const details: PaymentMethodDetails  =
```
Is it possible for `paymentMethodPlaceholders` to have any type other than `PaymentMethodDetails`? It looks like `paymentMethodPlaceholders` is defined on this page as a `PaymentMethodDetails`; do we need this conditional type here? 🤔 (Same comment applies to where this conditional type is used just below.)

Reply:
Ope, some leftovers after changes in https://github.com/Automattic/woocommerce-payments/pull/8964/commits/5f03d6c6a3675eb54fcf92d49f56512f67a91bd0. Thanks for noticing it! Fixed in ed8c70ec4da827a6e2ad4b10f40fe479ef3be073